locale/en/docs/guides/anatomy-of-an-http-transaction.md

```diff
@@ -126,7 +126,7 @@ request.on('error', function(err) {
 });
 ```
 
-There are other ways of [handling these errors][] such as using [`domains`][] or
+There are other ways of [handling these errors][] such as
```

Comment:
actually, that was another mistake - i committed directly, argh. should i revert the commit manually?